docs/site/tutorials/todo-list/todo-list-tutorial-model.md

```diff
@@ -85,7 +85,7 @@ export class TodoList extends Entity {
   // ...properties defined by the CLI...
 
   @hasMany(() => Todo)
-  todos?: Todo[];
+  todos: Todo[];
```

Comment:
I think this property is supposed to be optional, because `todos` is often not defined (e.g. when returned by `find` call and no inclusion was configured). I am proposing to revert this change.

Reply:
Ohh, I see, but this fix is just the docs was just to match the [code](https://github.com/strongloop/loopback-next/blob/verify-no-fk-required/examples/todo-list/src/models/todo-list.model.ts#L33). Should I change the actual code then?

Reply:
> I think this property is supposed to be optional, because todos is often not defined (e.g. when returned by find call and no inclusion was configured). I am proposing to revert this change.

+1

Reply:
Yes please, let's fix the code.